lib/screens/calender_screen/widgets/custom_card.dart

```diff
@@ -0,0 +1,141 @@
+import 'package:flutter/material.dart';
+
+class CustomCard extends StatelessWidget {
+  const CustomCard(this._color, this._opacityColor, {Key? key})
+      : super(key: key);
```

Comment:
don't use `this` with private variables. Making these named parameters would be an issue. 

Use constructor assignment like given this [example](https://dart.dev/guides/language/language-tour#constructors:~:text=Initializer%20lists%20are%20handy%20when%20setting%20up%20final%20fields.%20The%20following%20example%20initializes%20three%20final%20fields%20in%20an%20initializer%20list.%20Click%20Run%20to%20execute%20the%20code.)